.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,49 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
+{
+	"name": "Local Dockerfile",
+	"build": {
+		// Sets the run context to one level up instead of the .devcontainer folder.
+		"context": "..",
+		// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+		"dockerfile": "../Dockerfile",
+		"args": {
+			"PLUGIN": "geoips-clavrx"
+		  }
+	},
+	"remoteUser": "geoips_user",
+	"mounts": [
+		"source=/local/share/geoips/test_data,target=/mnt/geoips_testdata,type=bind,readonly",
+		"source=/mnt/data1/geoips_outdirs_temp,target=/output,type=bind",
+		"source=/home/gwynu/geoips/geoips_for_vscode,target=/app/geoips_packages/geoips,type=bind"
```

Comment:
Should this be generalized?

Reply:
Yeah, you're right. That should be generalized

Reply:
Fixed!